CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## NEXT
 
+### Changed
+* **BREAKING:** Accordion: Stateless ([#2130](https://github.com/dso-toolkit/dso-toolkit/issues/2130))
```

Comment:
Moeten we nog ergens duidelijk maken wat hieraan breaking is. Zijn wel een aantal dingen.

Reply:
Ik verwijs bij mijn aankondiging naar de bijbehorende blogpost. Ik ga nu DocumentComponent mergen en uitbrengen, ik zal ook in de CHANGELOG een verwijzing doen.